build/postinst

```diff
@@ -105,8 +106,8 @@ if [[ -e ${SETUP_DIR}/engines/$STOCKFISH ]]
 then
     if [[ -d $CENTAUR_ENGINE ]]
     then
-        mv ${CENTAUR_ENGINE}/stockfish_pi ${CENTAUR_ENGINE}/stockfish_pi.bak
-        mv ${SETUP_DIR}/engines/$STOCKFISH $CENTAUR_ENGINE
+#        mv ${CENTAUR_ENGINE}/stockfish_pi ${CENTAUR_ENGINE}/stockfish_pi.bak
+        mv ${SETUP_DIR}/engines/$STOCKFISH $CENTAUR_ENGINE/stockfish_$STOCKFISH_VERSION
```

Comment:
The commented line will backup original SF that comes with centaur. 
And the second one you need to keep the same name (stockfish_pi) so centaur software will find it. This way all games with centaur software will also end up in the database,

Reply:
I reverted my test changes that I did to do some manual testing with different engines on the centaur/engine folder.